pom.xml

```diff
@@ -18,20 +18,15 @@
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
 
-        <jenkins.version>2.375.4</jenkins.version>
-        <jenkins-tools-bom.artifactId>bom-2.375.x</jenkins-tools-bom.artifactId>
-        <jenkins-tools-bom.version>1887.vda_d0ddb_c15c4</jenkins-tools-bom.version>
+        <jenkins.version>2.414.3</jenkins.version>
+        <jenkins-tools-bom.artifactId>bom-2.414.x</jenkins-tools-bom.artifactId>
+        <jenkins-tools-bom.version>2582.v830625dd636c</jenkins-tools-bom.version>
 
         <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
 
         <aws-java-sdk.version>1.12.406-370.v8f993c987059</aws-java-sdk.version>
```

Comment:
Can be also removed when https://github.com/jenkinsci/jobcacher-plugin/pull/228 is merged

minimal is part of bom. Not full (https://github.com/jenkinsci/bom/blob/master/bom-weekly/pom.xml#L1002)